doc/release/release_0.18.rst

```diff
@@ -0,0 +1,418 @@
+- Replace words with racial connotations (#4921)
+- Fixes to apply_parallel for functions working with multichannel data (#4927)
+- Improve description of h_maxima and h_minima functions (#4928) (#4929)
+- CI: Skip doc build for PYTHONOPTIMIZE=2 (#4930)
+- MAINT: Remove custom fused type in skimage/morphology/_max_tree.pyx (#4931)
+- MAINT: remove numpydoc option, issue fixed in numpydoc 1.0 (#4932)
+- modify development version string to allow use with NumpyVersion (#4947)
+- CI: Add verbose option to avoid travis timeout for OSX install script  (#4956)
+- Fix CI: ban sphinx-gallery 0.8.0 (#4960)
+- Alias for data.chelsea: data.cat() (#4962)
+- Fix typo. (#4963)
+- CI: Use Travis wait improved to avoid timeout for OSX builds (#4965)
+- Small enhancement in "Contour finding" example: Removed unused variable n (#4967)
+- MAINT: remove unused imports (#4968)
+- MAINT: Remove conditional import on networkx (#4970)
+- forbid latest version of pyqt (#4973)
+- Remove warnings/explicit settings on feature, restoration (#4974)
+- Docstring improvements for label and regionprops_label (#4983)
+- try to fix timeout problem with circleci (#4986)
+- improve Euler number example (#4989)
+- [website] Standardize Documentation index page. (#4990)
+- Proofread INSTALL file. (#4991)
+- Catch leftover typos in INSTALL file. (#4992)
+- Let tifffile.imread handle additional keyword arguments (#4997)
+- Update docstring for random_noise function (#5001)
+- Update sphinx mapping for sklearn and numpy (#5003)
+- Update docstring slic superpixels (#5014)
+- Bump numpy versions to match scipy (kinda) (#5016)
+- Fix usage of numpy.pad for old versions of numpy (#5017)
+- [MRG] Update documentation to new data.camera() (#5018)
+- bumped plotly requirement for docs (#5021)
+- Fix IndexError when calling hough_line_peaks with too few angles (#5024)
+- Code simplification after latest numpy bump (#5027)
+- Fixes broken link to CODE_OF_CONDUCT.md (#5030)
+- Specify whether core dev should merge right after second approving review. (#5040)
+- Update pytest configuration to include ``test_`` functions (#5044)
+- MAINT Build fix for pyodide (#5059)
+- reduce OSX build time so that Travis is happy (#5067)
+- DOC: document the normalized kernel in prewitt_h, prewitt_v (#5076)
+- Some minor tweaks to CI (#5079)
+- removed usage of numpy's private functions from util.arraycrop (#5081)
+- peak_local_max: remove deprecated `indices` argument from examples (#5082)
+
+
+52 authors added to this release [alphabetical by first name or login]
+----------------------------------------------------------------------
+
+A warm thank you to all contributors who added to this release. A fraction of contributors were first-time contributors to open source and a much larger fraction first-time contributors to scikit-image. It's a great feeling for maintainers to welcome new contributors, and the diversity of scikit-image contributors is surely a big strength of the package. 
```

Comment:
May be reformat this to 80 characters length lines?

Reply:
@rfezzani I didn't want to go through another cycle of review just for minor formatting. There's actually quite a few lines to reformat to get the whole file in under 80c. I'll make this a PR between the RC and the actual release.